torch/testing/_internal/opinfo/definitions/special.py

```diff
@@ -61,6 +62,21 @@ def sample_inputs_i0_i1(op_info, device, dtype, requires_grad, **kwargs):
         yield SampleInput(t)
 
 
+def sample_inputs_hermite_polynomial_he(op, device, dtype, requires_grad, **kwargs):
+    yield from sample_inputs_elementwise_binary(
+        op, device, dtype, requires_grad, **kwargs
+    )
+    make_arg = partial(
+        make_tensor,
+        device=device,
+        requires_grad=requires_grad,
+    )
+    if dtype == torch.float32:
+        yield SampleInput(
+            make_arg((S,), dtype=dtype), make_arg((L, 1), dtype=torch.float16)
+        )
```

Comment:
This seems a bit hacky. Would be better as an opinfo test that checks one sample with different dtype inputs.

Reply:
Fair enuf